Resources/Prototypes/_Stalker_EN/Entities/Objects/Clothing/Head/T3/VityazHelmet.yml

```
@@ -8,3 +8,14 @@
     sprite: _Stalker_EN/Objects/Clothing/hats/Vityaz1Merc.rsi
   - type: Clothing
     sprite: _Stalker_EN/Objects/Clothing/hats/Vityaz1Merc.rsi
+
+- type: entity
+  parent: STClothingHeadHelmetTacticalT3Base
+  id: ClothingHeadHelmetVityazKushch
+  name: Kushch Helmet
```

```suggestion
  name: Vityaz 1F 'Kushch' Helmet
```
Just to keep naming similar with other helmets to reduce confusion.